src/main/java/io/vertx/blueprint/todolist/entity/Todo.java

```diff
@@ -0,0 +1,110 @@
+package io.vertx.blueprint.todolist.entity;
+
+/**
+ * Todo Entity
+ */
+public class Todo {
```

Comment:
use DataObject annotation which will generate a encoder/decoder to json making the Utils class obsolete